### PR TITLE
feat: add .editorconfig

### PR DESCRIPTION
**Pull request by TonybynMp4**

### CODE REVIEW

**Review by solareon — APPROVED** [review]

File: .github/sync.yml [hunk 1]
```diff
@@ -63,5 +63,5 @@ group:
     Qbox-project/safecracker
     Qbox-project/txAdminRecipe
   files:
-  - source: repo-template/.github/
-    dest: .github/
+  - source: repo-template/
+    dest: ./
```
Comment (D4isDAVID) [inline .github/sync.yml:67]:
```suggestion
  - source: repo-template/.github/
    dest: .github/
  - source: repo-template/.editorconfig
    dest: .editorconfig
```
I'm not sure how syncing the entire base folder works... that might be messy and lead to unexpected changes, maybe specifying the files separately is better?

**Review by D4isDAVID — COMMENTED** [review]
Tbh I'm not entirely sure about this; do we want `.editorconfig` on *all* the repos specified? Will we keep the same coding style for *all* these repos? (for example the npwd apps or forks)

File: .github/sync.yml (hunk 1 above)
Comment (TonybynMp4) [reply .github/sync.yml:67]: 🤷🏻 shouldn't do anything bad, it just copies the files to the root 🧐

**Comment by TonybynMp4** [comment]
> Tbh I'm not entirely sure about this; do we want `.editorconfig` on _all_ the repos specified? Will we keep the same coding style for _all_ these repos? (for example the npwd apps or forks)

Well yeah 🤔

File: .github/sync.yml (hunk 1 above)
Comment (BerkieBb) [reply .github/sync.yml:67]: Seems like a pointless change tbh, works already, why change it

**Review by mafewtm — APPROVED** [review]

**Pull request merged**